src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs

```diff
@@ -226,7 +226,7 @@ private static SourceText GenerateForTypeWithBuiltInConverter(ContextGenerationS
             {
                 SourceWriter writer = CreateSourceWriterWithContextHeader(contextSpec);
 
-                string typeFQN = typeMetadata.TypeRef.FullyQualifiedName;
+                string typeFQN = typeMetadata.TypeRef.ToGlobalDisplayString();
```

Comment:
I couldn't find where this method is defined. Is it just concatenating `global::` to the FQN?